internal/cobraext/const.go

```diff
@@ -63,6 +63,9 @@ const (
 	SkipPullRequestFlagName        = "skip-pull-request"
 	SkipPullRequestFlagDescription = "skip opening a new pull request"
 
+	SkipTLSVerifyFlagName        = "skip-tls-verify"
```

Comment:
Nit. Use `tls` as prefix of the flag in case there are more tls flags in the future.
```suggestion
	SkipTLSVerifyFlagName        = "tls-skip-verify"
```

Reply:
Brain fart on my side, you're absolutely right about this. Working on this...